api/docs/release.dox

```diff
@@ -136,6 +136,7 @@ clients.
 The changes between version \DR_VERSION and 7.1.0 include the following minor
 compatibility changes:
 
+ - Added the type struct dr_zmm_t. dr_zmm_t is not used or supported yet.
```

Comment:
Adding a type is not a compatibility change, so it should not be in this list: it should be in the list of new features.

"not used or supported yet": so maybe it should not be mentioned here at all yet.

Reply:
See comment above.